docs/book/src/developer/guide.md

```diff
@@ -25,6 +25,8 @@ You'll need the [docker daemon][docker] v19.03 or newer available.
 
 [docker]: https://docs.docker.com/install/
 
+On MacOS systems using [Lima](https://github.com/lima-vm/lima) is a viable option to Docker Desktop.
```

Comment:
```suggestion
On MacOS systems, using [Lima](https://github.com/lima-vm/lima) is a viable alternative to Docker Desktop.
```
nit. Option sounds like it's a way to get Docker Desktop. Not sure about the `,`